dev/com.ibm.ws.security.openidconnect.clients.common/src/com/ibm/ws/security/openidconnect/clients/common/OIDCClientAuthenticatorUtil.java

```diff
@@ -486,4 +489,17 @@ static String extractIssuerFromTokenEndpointUrl(ConvergedClientConfig clientConf
         return issuer;
     }
 
+    private void setThreatClientId(String clientID) {
+        if (threadClientID == null) {
+            threadClientID = new ThreadLocal<String>();
+        }
```

Comment:
Should never happen since you initialize `threadClientID` when you declare the static variable.